plugins/foreman_openscap/1.0/index.md

```diff
@@ -0,0 +1,630 @@
+* [smart_proxy_openscap](https://github.com/theforeman/smart_proxy_openscap) - Smart-Proxy plugin which validates SCAP content from Foreman, distributes SCAP content to clients and posts ARF reports from client hosts to Foreman
+* [foreman_scap_client](https://github.com/theforeman/foreman_scap_client) - A client script which runs OpenSCAP scan and uploads the scan report to the Smart-Proxy
+* [puppet-foreman_scap_client](https://github.com/theforeman/puppet-foreman_scap_client) - A puppet module which configures foreman_scap_client
+* [ansible-foreman_scap_client](https://github.com/theforeman/ansible-foreman_scap_client) - Ansible role which configures foreman_scap_client, an alternative to puppet-foreman_scap_client
+
+## 1.1 OpenSCAP basic concepts
+
+There are four basic concepts (entities) in OpenSCAP plug-in: SCAP Contents, Compliance Policies, ARF Reports and Tailoring Files.
+
+__SCAP Content__ represents SCAP DataStream XML file as defined by SCAP 1.2 standard. DataStream file contains implementation of compliance,
+configuration or security baselines. Users are advised to acquire examplary baseline by installing scap-security-guide package.
+DataStream file usualy contains multiple XCCDF Profiles. Each for different security target. The content of DataStream file can be inspected by oscap tool from openscap-scanner package.
+(XCCDF = Extensible Configuration Checklist Description Format, XCCDF profile = A checklist which audit specific security target)
+
+__Tailoring File__ is a XML file very much like SCAP Content. It represents a customization of compliance defined in a SCAP Content XML file. Additional details and instructions how to create such a file are available in the official openscap [documentation](https://www.open-scap.org/resources/documentation/customizing-scap-security-guide-for-your-use-case/).
+
+__Compliance Policy__ is high level concept of a baseline applied to the infrastructure. Compliance policy is defined by user on web interface. Users may assign following information to the policy:
+
+* Deployment Options - how the client will be deployed. The options are Ansible (provided you have foreman_ansible plugin), Puppet or manual. It is recommended to use Puppet or Ansible for client deployment as they provide a convenient way of applying changes. See section [2.3](plugins/foreman_openscap/0.9/index.html#2.3Policydeploymentoptions) for details on policy deployment options.
+* SCAP Content
+* XCCDF Profile from particular SCAP Content
+* Tailoring File
+* XCCDF Profile from a Tailoring File
+* Hostgroups that should comply with the policy
+* Schedule - the period in which the audit shall occur
+
+__ARF Report__ is XML output of single scan occurrence per single host. Asset Reporting File format is defined by SCAP 1.2 standard. Foreman plug-in stores the ARF Reports in database for later inspections.
+
+
+## 1.2 Release notes
+
+Compatibility matrix
+
+<table class="table table-bordered table-condensed">
```

Comment:
This probably needs to be updated as well, i don't think anyone cares about 0.3.4 being compatible with foreman 1.7 when they read the manual for 1.0 version of the plugin